v2/cf-nodejs-logging-support/src/lib/config/config.ts

```diff
@@ -0,0 +1,143 @@
+import { MergedConfig, ConfigObject, ConfigField, customFieldsFormat } from './interfaces';
+import * as coreConfig from './config-core.json';
+import * as requestConfig from './config-request.json';
+import * as cfConfig from './config-cf.json';
+import * as kymaConfig from './config-kyma.json';
+import * as appLoggingConfig from './config-app-logging.json';
+import * as cloudLoggingConfig from './config-cloud-logging.json';
+import EnvService from '../core/env-service';
+
+export default class Config {
+
+    private static instance: Config;
+
+    private config: MergedConfig = {
+        "fields": [],
+        "customFieldsFormat": "cloud-logging",
+        "outputStartupMsg": false
```

Comment:
could be split up, as there are two concerns done here:
-  setting of the library itself
- configuration of the logging field behaviour

might get convoluted in the future, if there are more environmental global settings added.

Reply:
Why would more global settings make the config convoluted? Field configs are separated from additional configs. 

The idea is to have a single configuration singleton internally, while externally you can split/extend configuration properties into multiple files if required. This means, e.g., `outputStartupMsg` could be overwritten in a different config file than the `fields` configuration. In my opinion "fields" is just another setting of the library itself besides other configs.

Reply:
then just more about the naming of this config object.
It is concerned with more then the merged config, or i would call the config object for field settings "merged config"

If the field settings are clear, and the whole configuration set is called config, i am fine with that approach,
but would like to have field settings not being refered to as config as well then, as this convolutes the levels on which things are handled

Reply:
Can also be improved another time, so do not take this one as a merge blocker